exercises/change/example.js

```diff
@@ -0,0 +1,139 @@
+class Change {}
+
+export default Change;
+
+// data structure to hold each candidate solution that is generated
+class Candidate {
+  constructor() {
+    this.wasSearched = false;
+    this.coins = [];
+  }
+
+  searched() {
+    this.wasSearched = true;
+  }
+
+  isSearched() {
+    return this.wasSearched;
+  }
+
+  getCoins() {
+    return this.coins;
+  }
+
+  addCoin(coin) {
+    const sortNum = (a, b) => a - b;
+
+    this.coins.push(coin);
+    this.coins.sort(sortNum);
+  }
+
+  getCoinCount() {
+    return this.coins.length;
+  }
+
+  getSum() {
+    const getSum = (total, num) => total + num;
+    return this.coins.reduce(getSum);
+  }
+}
+
+Change.prototype.calculate = (coinArray, target) => {
```

Comment:
In ES6 we would want to see the `Change` class written something like this:

```javascript

class Change {
  calculate(coinArray, target) {
    ...
  }
}
```

The preferred pattern is like you have done for your `Candidate` class.  The pattern of extending the prototype is an ES5 pattern, so we'll want to use the newer paradigm here.